### PR TITLE
update documentation files

### PR DESCRIPTION
**Pull request by codingUniv**

### CODE REVIEW

File: document/ug/cfgprm.tex [hunk 1]
```diff
@@ -0,0 +1,21 @@
+\begin{table}[H]
```
Comment (jjts) [inline document/ug/cfgprm.tex:1]: the only file that can be added is custom.tex 

you cannot add other files as you cannot modify ug.tex

Comment (jjts) [reply document/ug/cfgprm.tex:1]: same comment for regs.tex 

However, if your custom.tex includes these files it should be ok

**Comment by jjts** [comment]
Please see the review -- this PR can be merged after you add custom.tex 
is 

make document DOC=ug 

finishing OK ?

**Comment by codingUniv** [comment]
For the submitted updates, make doc-build DOC=ug is OK

**Pull request merged**